### PR TITLE
Adding libsuitesparseconfig to debian install [Backport]

### PR DESCRIPTION
**Pull request by AnHeuermann**
Updating packages for OpenModelica/OpenModelica#7151

### CODE REVIEW

**Comment by adrpo** [comment]
Pattern `debian/tmp/usr/lib/*/omc/lib*amd*` will also cover: `debian/tmp/usr/lib/*/omc/libcolamd*`

**Comment by adrpo** [comment]
Just leave one of them

File: debian/libomcsimulation.install [hunk 1]
```diff
@@ -12,14 +12,18 @@ debian/tmp/usr/lib/*/omc/libzlib*
 debian/tmp/usr/lib/*/omc/libOpenModelicaFMIRuntimeC*
 debian/tmp/usr/lib/*/omc/libSimulationRuntimeC.*
 debian/tmp/usr/lib/*/omc/libSimulationRuntimeFMI*
+## SuiteSparse
+debian/tmp/usr/lib/*/omc/lib*amd*
 debian/tmp/usr/lib/*/omc/libbtf*
+debian/tmp/usr/lib/*/omc/libcolamd*
```
Comment (adrpo) [inline debian/libomcsimulation.install]: Remove this one as is not needed.

**Comment by AnHeuermann** [comment]
Okay, I updated the PR

**Review by adrpo — APPROVED** [review]
LGTM!

File: debian/libomcsimulation.install (hunk 1 above)
Comment (AnHeuermann) [reply debian/libomcsimulation.install]: It should already be removed

**Pull request merged**